### ISSUE TITLE
Exporting an alias to a CSS module breaks react-refresh and causes full page refresh

### ISSUE THREAD

**Issue by Zaid-Ajaj** [pmmmwh/react-refresh-webpack-plugin#260]
I have the following code which works perfectly with react-refresh: changing either CSS or JS doesn't fully refresh the page and instead only re-renders the components:
```js
import stylesheet from "./styles/counter.css";

export const Counter = () => {
  const [count, setCount] = useState(0)
  return createElement("div", null, 
    createElement("h1", null, count),
    createElement("button", { onClick: () => setCount(count + 1) }, "Increment")
  );
}
```
However, once I introduce an exported alias like the following, it no longer picks up hot updates and instead refreshes the entire page
```js
import stylesheetA from "./styles/counter.css";

export const stylesheetB = stylesheetA

export const Counter = () => {
  const [count, setCount] = useState(0)
  return createElement("div", null, 
    createElement("h1", null, count),
    createElement("button", { onClick: () => setCount(count + 1) }, "Increment")
  );
}
```
I will be trying to make a full repro with this examples but this is basically the gist of it. I would really appreciate it if you could guide me into debugging such issues. I am trying to add first class react-refresh support to [Fable](https://fable.io) application but due to some generated artifacts (like the one above) the experience is affected. 

Using latest goodies:
 - react-refresh v0.9.0
 - @pmmmwh/react-refresh-webpack-plugin v0.4.3
 - react v16.14.0
 - webpack v4.44.2

**Comment by pmmmwh** [comment]
I think this have something to do with us having to bail out when we cannot statically determine if all exports are side-effect free. When we cannot establish a boundary we would propagate the update to the parent component, and if this propagation reaches root it will cause a full refresh.

**Comment by pmmmwh** [comment]
Fixed in `0.5.0-beta.1`.

**Comment by Zaid-Ajaj** [comment]
Thanks a lot! @pmmmwh 🙏
